packages/compass-generative-ai/src/components/ai-experience-entry.tsx

```diff
@@ -17,7 +18,7 @@ import {
 } from './ai-entry-svg';
 
 const hiddenOnNarrowStyles = css({
-  [`@container compass-workspace-container (width < 900px)`]: {
```

Comment:
Do we want then to change the other container query to use this for consistency?